server/src/test/resources/config/sidecar_cdc.yaml

```
@@ -58,6 +58,8 @@ sidecar:
       max_execution_time: 15m # 15 minutes
   cdc:
     segment_hardlink_cache_expiry: 1m # 1 minute
+    is_enabled: true
```

for consistency let's use `enabled` instead. Also this should be included in `conf/sidecar.yaml` with proper documentation
```suggestion
    enabled: true
```